cmd/launch_producer/main.cpp

```diff
@@ -66,8 +67,29 @@ const char* kProducerPaths[] = {
 std::string getDriver() {
   std::string driver;
 
-  GET_STRING_PROP(kDriverProperty, driver);
-  LOG_INFO("Driver package: %s", driver.c_str());
+  // Read gapid.driver_package_override setting to override default property.
```

Comment:
move this part to its own function `attemptToOverrideDriverPackage`.

Reply:
Done

Reply:
Done